packages/editor/src/components/debug-menu/debug-menu.ts

```diff
@@ -3,12 +3,12 @@ import { customElement, property, state } from 'lit/decorators.js';
 
```

Comment:
Why not just use `components/debug-menu.ts`

Reply:
I think  as the complexity of the features goes up, debug-menu may will be split by some atom compontents, so put it in a file is fine.